extension/opampcustommessages/metadata.yaml

```diff
@@ -0,0 +1,3 @@
+status:
+  codeowners:
+    active: [BinaryFissionGames, evan-bradley]
```

Comment:
@evan-bradley I've put you down as a codeowner for this component

Reply:
Fine with me. Thanks!